### PR TITLE
feat: add Totano to ingredients taxonomy

### PR DESCRIPTION
**Pull request by VaiTon**

### CODE REVIEW

**Review by aleene — CHANGES_REQUESTED** [review]
Can you add the english name as well en:European flying squid and the latin name.
On the wikidata page you can also link back to OFF by adding a parameter there.

**Comment by VaiTon** [comment]
@aleene on wikidata I only see "OpenFoodFacts category". Is there another one for ingredients?

**Comment by aleene** [comment]
Search for open food (with the space)

**Review by aleene — CHANGES_REQUESTED** [review]
Language code is la: not lat:

**Comment by aleene** [comment]
Wonder is we should use the latin name for all fish entries.

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -71065,9 +71071,11 @@ ciqual_food_code:en:10001
 ciqual_food_name:en:Squid, raw
 ciqual_food_name:fr:Calmar ou calamar ou encornet, cru
 
-<en:squid
-en:Teuthida
-la:Teuthida
+<en:Teuthida
+en:european flying squid
+it:totano
+lat:todarodes sagittatus
```
Comment (VaiTon) [inline taxonomies/ingredients.txt:71077]:
```suggestion
la:todarodes sagittatus
```

**Review by aleene — APPROVED** [review]

**Pull request merged**